provisioner_test.go

```diff
@@ -119,3 +123,75 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	l := log.New(log.Writer(), log.Prefix(), log.Flags())
+
+	appName := "Test"
+	dataDir, _ := filepath.Abs("./testdata/tmp")
```

Comment:
you should use `mockedTempPath()` instead

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/ae08b36c5a8f4b2b68add1fd5808ad5e8201c78a#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR124-R126